coral/README.md

```diff
@@ -46,4 +46,4 @@ Here we list all explorers, wallets, and apps you can access with a browser:
 
 * [Wasm Glass](https://demonet.wasm.glass) - contract explorer
 * [Name App](https://cosmwasm.github.io/name-app/) - Sample dApp interface to use the 
-  [nameservice contract](https://github.com/CosmWasm/cosmwasm-examples/tree/nameservice-0.5.2/nameservice) 
\ No newline at end of file
+  [nameservice contract](https://github.com/CosmWasm/cosmwasm-examples/tree/nameservice-0.5.2/nameservice) 
```

Comment:
This is now 0.6.0. I will update and merge

Reply:
The diff was probably just created because of a missing newline. I did not touch this